app/helpers/application_notifications_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ApplicationNotificationsHelper
+  # Returns the view path of the notification
+  #
+  # @param [Course::Notification|UserNotification] notification The target notification
+  # @return [String] The view path of the notification
+  def notification_view_path(notification)
+    activity = notification.activity
+    root_path = "#{activity.notifier_type}/#{activity.event}"
+    notification_type = notification.class.name.underscore.sub('/', '_').pluralize
```

Comment:
cuz, it will get `course/notification` but we want `somepath/course_notification/xxx`